src/main/scala/mesosphere/mesos/TaskBuilder.scala

```diff
@@ -196,7 +216,7 @@ class TaskBuilder(app: AppDefinition,
   protected def computeContainerInfo(ports: Seq[Int]): Option[ContainerInfo] = {
     if (app.container.isEmpty && app.ipAddress.isEmpty) None
     else {
-      val builder = ContainerInfo.newBuilder
+      var builder = ContainerInfo.newBuilder
```

Comment:
builder used to be assigned to the result of the injector, looks like that was lost in the refactor. not a big deal as long as we want to acknowledge (officially) that injection has side effects and mutates the context parameter.